CHANGELOG.md

```diff
@@ -1,5 +1,71 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.10.7
+
+<!-- generated comparing v5.10.6..master -->
+
+_Sep 26, 2022_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 [Divider](https://mui.com/joy-ui/react-divider/) component is added to Joy UI (#34403) @siriwatknp
+
+### `@mui/material@5.10.7`
+
+- [CssVarsProvider] Exclude dark mode variables from `:root` stylesheet (#34131) @siriwatknp
+- [Chip] Add chip classes (#33801) @pratikkarad
+- [Slider] Fix typo in the comments in the source (#34452) @HexM7
+- [SvgIcon] Fix passing an ownerState to SvgIcon changes the font size (#34429) @ZeeshanTamboli
+- [Stepper] Fix optional label is not centered when `alternativeLabel` is used (#34335) @ZeeshanTamboli
+- [Tooltip] Add undefined, null or false in `title` (#34289) @abhinav-22-tech
+
+### `@mui/system@5.10.7`
+
+- [system] Fix parsing of hsla colors in getLuminance (#34437) @ptrfrncsmrph
+- [system] Fix incorrect type of `shape.borderRadius` in theme (#34076) @ZeeshanTamboli
+- [system] Replace `enableSystem` with `defaultMode` (#33960) @siriwatknp
+
+### `@mui/base@5.0.0-alpha.99`
+
+#### Breaking changes
+
+- [TrapFocus] Rename TrapFocus to FocusTrap (#34216) @kabernardes
+
+  ````diff
+  - import TrapFocus from ‘@mui/base/TrapFocus’;
+  + import FocusTrap from ‘@mui/base/FocusTrap’;    ```
+  ````
+
+### `@mui/joy@5.0.0-alpha.47`
+
+- [Checkbox][joy ui] spread `value`, `required`, and `readOnly` to input (#34477) @siriwatknp
+- [Joy][chip] Fix unbinded `onClick` prop (#34455) @HexM7
+- [Joy] Add `Divider` component (#34403) @siriwatknp
+- [Radio][joy ui] spread `readOnly` and `required` to input (#34478) @siriwatknp
```

Comment:
Can we unify the tags we use?

Reply:
Updated.

Reply:
I was thinking that we should use [Component][Product] with the same casing, but let's resolve it separately.